src/main.cpp

```diff
@@ -2299,6 +2299,11 @@ bool ConnectBlock(const CBlock& block, CValidationState& state, CBlockIndex* pin
         nLockTimeFlags |= LOCKTIME_VERIFY_SEQUENCE;
     }
 
+    // Start enforcing WITNESS rules using versionbits logic.
+    if (IsWitnessEnabled(pindex->pprev, chainparams.GetConsensus())) {
```

Comment:
Just a reminder to update the commit message for this commit, to reflect versionbits usage (rather than v5 blocks).